### ISSUE TITLE
Support x-delayed-message feature!

### ISSUE THREAD

**Issue by eldar-fairfly** [mosquito/aio-pika#44]
It will be extremely helpful if you can add `x-delayed-message`  to `ExchangeType` Enum class.
Currently it's impossible to create such an exchange.

```python
@unique
class ExchangeType(Enum):
    FANOUT = 'fanout'
    DIRECT = 'direct'
    TOPIC = 'topic'
    HEADERS = 'headers'
    X_DELAYED_MESSAGE = 'x-delayed-message'
```

**Comment by mosquito** [comment]
Thank you for idea. A already uploaded it as `0.15.1`

**Comment by eldar-fairfly** [comment]
You welcome :)
@mosquito when is the new fix will be released? the current released version is: `0.6.2`
